app/posts/manage-teacher-training-applications/2021-02-11-improving-the-activity-log-and-timeline.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Improving the activity log and timeline
+description: Some improvements to the activity log and timeline driven by the need accomodate interviews that can be changed or cancelled
+date: 2021-03-03
+---
+
+When we designed the [interview journeys](/manage-teacher-training-applications/interviews-iteration-3/) we realised the activity log and timeline need to handle events relating to things that can be changed or deleted.
+
+As a minimal solution at the time, we decided to hide the ‘view interview’ link on any event relating to an interview which no longer exists.
+
+The problem is that without a link users cannot see the details of the interview and so cannot fully understand what’s happened.
+
+We decided also to look at how notes are displayed in the activity log and timeline. Currently notes appear only as a link, meaning that users always need to click through to read the note.
+
+## What we did
+
+We added the details of interviews and notes so that users can understand what’s happened without leaving the activity log or timeline.
+
+We also moved away from the blue ‘train track’ styling which didn’t work very well, especially on the activity log where it implied a connection between events in different applications.
+
+It also sets us up nicely to use the same application list layout when we introduce search and filter.
```

Comment:
I wasn't sure what you meant here. Could you explain a little further?

Reply:
The left hand side filter that may likely end up there, works better with the new (non train track) styles visually and is consistent with the application list styles of simple horizontal rules split up by h2s.